### ISSUE TITLE
lsb0 Breaks Bitshifting

### ISSUE THREAD

**Issue by hmorris94** [scott-griffiths/bitstring#251]
When `lsb0` is False, this operation behaves as expected. (Actually better than expected in that it preserves the array size!)
```python
a = bitstring.BitArray(bin="11001")

(a << 1).b
'10010'
(a >> 1).b
'01100'
```

When `lsb0` is True, behavior is a little funny.
```python
(a << 1).b
'11000'
(a >> 1).b
'01001'
```

**Comment by scott-griffiths** [comment]
Yep, that behaviour is definitely a little funny! The behaviour for << and >> shouldn't depend on the value of the lsb0 flag at all.

Should be an easy fix along with a few other lsb0 fixes I have in progress. Many thanks for bringing it to my attention.

**Comment by scott-griffiths** [comment]
Fixed in 4.0.2
